runtime-sdk/src/crypto/signature/sr25519.rs

```diff
@@ -31,7 +32,7 @@ impl PublicKey {
         Ok(PublicKey(bytes.to_vec()))
     }
 
-    /// Verify a signature.
+    /// Verify a signature used in Oasis SDK transactions.
```

Comment:
Should we compute the digest and call `verify_raw` or `verify_transcript/digest`, to avoid code duplication?